lib/template/datepicker.template.js

```diff
@@ -30,7 +30,7 @@ export default () => `
         PM
       </label>
     </div>
-
+    <div class="c-datepicker-mask"></div>
```

Comment:
This classname should be `c-datepicker__mask` (according to [BEM](http://getbem.com/introduction/)).

Can you explain what this element is for? It isn't obvious to me, and I can't tell what different removing it makes.